README.md

```diff
@@ -1,18 +1,70 @@
-Longhorn Upgrade Responder
+Upgrade Responder
 ========
 
-## Building
+## Overview
+Upgrade Responder enables developers to know how many users are using their applications. 
+This project is GDPR compliant since it doesn't store any information that can identify the user, not even IP address.
 
-`make`
+For example, longhorn use Upgrade Responder project to create the public metric dashboard at [https://metrics.longhorn.io/](https://metrics.longhorn.io/)
 
+## Usage
+The overall idea is that application developers modify their application to periodically send requests that contain current app version and Kubernetes version to the Upgrade Responder server.
+The Upgrade Responder will extract that information and store it into InfluxDB.
+The Upgrade Responder will also return a response that contains the latest version of the application.
+A Grafana dashboard will pull the measurements out of InfluxDB and draw statical graphs.
+The steps are detailed below:
 
-## Running
 
-`./bin/upgrade-responder --debug start --upgrade-response-config /etc/upgrade-responder/upgrade-response.json --influxdb-url http://localhost:8086`
+### 1. Running Upgrade Responder server
+Use the image `longhornio/upgrade-responder` with the starting command:
+```
+upgrade-responder --debug start <FLAGS>
+```
+The available flags are:
 
-It will listen on `0.0.0.0:8314`
+| Flags  | Example value  | Description  |
+|---|---|---|
+| `--upgrade-response-config` | `/etc/upgrade-responder/upgrade-response.json` | Specify the response configuration file for upgrade query. The Upgrade Responder server uses this file to determine the latest version of the application. See [References](#references) for an example of a configuration file  |
+| `--application-name` | `awesome_app` | Specify the name of the application that is using this Upgrade Responder server. This will be used to create a database named `<application-name>_upgrade_responder` in the InfluxDB to store all data for this Upgrade Responder |
+| `--influxdb-url` | `http://localhost:8086` | Specify the URL of InfluxDB. Note that we currently only support InfluxDB version 1.8 and before  |
+| `--influxdb-user` | `admin` | Specify the InfluxDB username |
+| `--influxdb-pass` | `password` | Specify the InfluxDB password |
+| `--influxdb-continuous-query-period` | `1h` | Specify the period for InfluxDB Continuous Queries. This value is used to aggregate data every PERIOD interval. This value should not be changed after it is set the first time. The default value is 1h. This value should be the same as time in GROUP BY clause in Grafana. See [References](#references) for more details |
+| `--geodb` | `/etc/upgrade-responder/GeoLite2-City.mmdb` | Specify the path of to GeoDB file.  See [References](#references) for more details about GeoDB |
+| `--port` | `8000` | Specify the port number. By default port `8314` is used |
+
```

Comment:
How to verify the server is correctly up and running?

Add `curl -d <request> <endpoint>` to verify the server.

Also, add how to verify the request was correctly recorded in the influxdb. Including an example of the record to show all the information we've stored.

Also noted `curl -d {} <endpoint>` as a quick way to check the server.